media-tv/kodi/kodi-9999.ebuild

```diff
@@ -234,28 +234,27 @@ src_configure() {
 		$(use_enable X x11)
 }
 
-src_compile() {
-	emake V=1
```

Comment:
I'd rather stick to the default src_compile... is there a reason for this to be verbose beyond Gentoo's defaults?